calico-policies/private-network-isolation/eu-es/allow-private-services-pods.yaml

```diff
@@ -0,0 +1,84 @@
+# This policy allows outgoing network traffic from pods to other
+# IBM Cloud services that support private service endpoints.
+# The example nets in this policy are for IBM Cloud Container Registry,
+# IBM Cloud Monitoring with Sysdig, and IBM Log Analysis with LogDNA.
+
+apiVersion: projectcalico.org/v3
+kind: GlobalNetworkPolicy
+metadata:
+  name: allow-private-services-pods
+spec:
+  egress:
+  - action: Allow
+    destination:
+      nets:
+      # IBM Cloud Container Registry: Global (US East) registry subnets
+      - 166.9.251.49/32
+      - 166.9.251.82/32
+      - 166.9.251.113/32
+      # IBM Cloud Container Registry: EU Central registry subnets
+      - 166.9.248.76/32
+      - 166.9.248.105/32
+      - 166.9.248.136/32
+      ports:
+      - 443
+      - 4443
+    protocol: TCP
+    source: {}
+  - action: Allow
+    destination:
+      nets:
+      # IBM Cloud Monitoring with Sysdig: eu-de
+      # Taken from: https://cloud.ibm.com/docs/monitoring?topic=monitoring-endpoints
+      - 166.9.28.71/32
```

Comment:
Do you know if sysdig and/or logDNA have (or will have) separate IPs for eu-es (separate from the eu-de ones)?  Just because registry doesn't have separate eu-es IPs, doesn't mean logDNA or sysdig will necessarily do the same thing.

Reply:
Yes, I confirmed this as of now they are using eu-de endpoints for logDNA and Sysdig and will change in the future.